### ISSUE TITLE
Cannot read property 'replace' of undefined

### ISSUE THREAD

**Issue by MaienM** [onury/docma#41]
When trying to view the generated documentation of one of our projects I get a blank page and the following error in the console: 

```
docma-web.js:32 Uncaught TypeError: Cannot read property 'replace' of undefined
    at Object.helper.getSymbolInfo (app.min.js:17)
    at getSymbolData (app.min.js:17)
    at getSidebarNavItem (app.min.js:17)
    at app.min.js:17
    at Array.forEach (<anonymous>)
    at Object.helper.buildSidebarNodes (app.min.js:17)
    at app.min.js:17
    at Array.forEach (<anonymous>)
    at Object.helper.buildSidebarNodes (app.min.js:17)
    at app.min.js:17
helper.getSymbolInfo @ app.min.js:17
getSymbolData @ app.min.js:17
getSidebarNavItem @ app.min.js:17
(anonymous) @ app.min.js:17
helper.buildSidebarNodes @ app.min.js:17
(anonymous) @ app.min.js:17
helper.buildSidebarNodes @ app.min.js:17
(anonymous) @ app.min.js:17
dust.filter @ docma-web.js:4
Chunk.reference @ docma-web.js:4
body_0 @ docma-web.js:20
load @ docma-web.js:4
Chunk.partial @ docma-web.js:4
body_1 @ docma-web.js:13
Chunk.exists @ docma-web.js:4
body_0 @ docma-web.js:13
load @ docma-web.js:4
dust.render @ docma-web.js:4
DocmaWeb._render @ docma-web.js:32
DocmaWeb.render @ docma-web.js:32
DocmaWeb.Route.apply @ docma-web.js:34
(anonymous) @ docma-web.js:43
setTimeout (async)
(anonymous) @ docma-web.js:43
(anonymous) @ docma-web.js:9
o @ docma-web.js:9
(anonymous) @ docma-web.js:9
o @ docma-web.js:9
L.dispatch @ docma-web.js:9
L.replace @ docma-web.js:9
L.start @ docma-web.js:9
window.onload @ docma-web.js:43
load (async)
(anonymous) @ docma-web.js:43
(anonymous) @ docma-web.js:43
```

Looking into the file, this seems to be due to the `kind.replace` call. Replacing this with `(kind || "unknown").replace` resolves the issue.

It appears this problem is caused by having a (non-exported) documented top-level variable. For example:

``` javascript
/**
 * This is a constant.
 */
const CONSTANT = 42;
```

**Comment by jayesbe** [comment]
```
/**
 * Server Error
 *
 * @module av/error
 */
export default function ServerError(message) { 
  this.message = message; 
  this.stack = (new Error()).stack; 
} 
ServerError.prototype = new Error; 
ServerError.prototype.name = "ServerError";
ServerError.prototype.constructor = ServerError;

```

```
  "src": [
    {
      "mylib": "./src/server-error.js"
    },
    {
      "readme": "./README.md"
    }
  ],
```

produces the same error

**Comment by onury** [comment]
Fixed with [Docma v3.0.0](https://onury.io/docma/changelog).
